BrainPortal/public/swagger/cbrain-4.5.0-swagger.json

```diff
@@ -0,0 +1,204 @@
+{
```

Comment:
Yes, that's the point. It's the description for THIS version of CBRAIN. We'll change the filename and keep them in sync as needed. But for the moment, anyway, we're still working on the first one.